src/components/OtherGlossary.jsx

```diff
@@ -0,0 +1,195 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { styled } from "@stitches/react";
+
+import GlossaryList from "./GlossaryList";
+import Button from "./shared/Button";
+import ContainerStyled from "./shared/Container";
+import TabContainer from "./shared/TabContainer";
+import Title from "./shared/Title";
+import SubTitle from "./shared/SubTitle";
+import ErrorStyled from "./shared/Error";
+
+import chromeStore from "../utils/chromeStore";
+import { convertObjectToCsv } from "../utils/convert";
+import {
+  createGlossaryFromGoogleTranslation,
+  getGlossaryFromServer,
+  updateCsvFromGoogleStorage,
+  updateGlossaryFromServer,
+} from "../services/glossaryService";
+
+const GlossaryListWrap = styled("div", {
+  flex: 1,
+});
+
+function OtherGlossary() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [errorMassage, setErrorMassage] = useState(null);
+  const [myGlossary, setMyGlossary] = useState(null);
+  const [otherGlossary, setOtherGlossary] = useState(null);
+  const [isServiceAccess, setIsServiceAccess] = useState(null);
```

Comment:
```suggestion
  const [hasServiceAccess, setHasServiceAccess] = useState(null);
```

서비스 접근 권한을 갖고 있다는 의미로 has를 붙이는 게 더 좋아보여요!